src/components/NavigationMailbox.vue

```diff
@@ -679,3 +680,9 @@ export default {
 	max-width: initial;
 }
 </style>
+
+<style lang="scss">
+.subMailBoxCounter .app-navigation-entry__counter-wrapper {
```

Comment:
`.app-navigation-entry__counter-wrapper` is a selector for an internal element of AppNavigationItem. This works, but it's fragile. Internal restructuring of the component may break this CSS rule.

Could we fix this upstream in `@nextcloud/vue`?

Positive side effect: the issue will also be fixed for any other apps that use AppNavigationItem with a counter.